util.go

```diff
@@ -19,6 +19,14 @@ func renameWithFallback(src, dest string) error {
 		return err
 	}
 
+	// Windows cannot use syscall.Rename to rename a directory
```

Comment:
Right, but that doesn't say it fails when attempting to rename a directory, period - just that it fails if attempting to rename a directory to an existing empty directory. At this point, we haven't established yet whether the target dir exists, or (if it does) whether it's empty.